rfcs/accessibility/accessibility.md

```diff
@@ -0,0 +1,74 @@
+# RFC: Accessibility
+
+---
+
+Contributors:
+@jurokapsiar
+
+## Summary
+
+Fluent UI components are designed to support various aspects of accessibility, so that they can be used with different input methods (mouse, touch, keyboard, screen readers) as well as fulfill different rendering and layout requirements (theming, zoom, contrast).
+
+## Background
+
+[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/) and [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.2/) are community driven standard for accessible web pages and applications. They provide basic framework allowing consistent usage of web pages and applications with or without assistive technologies.
+
+## Problem statement
+
+Fluent UI components aim to fully respect both WCAG and ARIA and the Fluent UI team is working with the community to improve them. From the technical point of view, thee main challenges were identified:
+
+1. Accessibility is usually not intuitive as it is not immediately obvious. It is hard to achieve high quality user experience and easy to regress it.
+2. Each screen reader and browser works differently, often making it hard or even impossible to achieve high quality user experience for all combinations. Browsers and screen readers release new versions that change the experience
+3. Not all UI/UX designs can be sufficiently represented with ARIA roles and patterns
+
+## Detailed Design or Proposal
+
+To address these challenges, following principles will be followed:
+
+- Challenge 1: Sufficient test coverage with understandable and documented tests
+- Challenge 2: Components will be created based on well tested prototypes or existing components. We will report and track issues to the screen readers vendors
+- Challenge 3: We will work with the community to improve standards. We will maintain set of best practices and validations that help consumers design accessible user interfaces
+
+### Target accessibility areas
+
+- DOM structure that provides semantical value either by using correct element types or roles
+- ARIA attributes that are valid for elements/roles used and provide correct state information about the component or element
+- Keyboard navigation (navigation by tabbing, arrow keys, pagination or letter keys, click and right click (Enter/Space, Shift+F10), and close (Escape))
```

Comment:
Maybe I would just add something like. 
Keyboard navigation is applied based on component.  (In order to avoid confusion that in every component has the same keyboard navigation expectation)